pkg/syncer/status/status_controller.go

```diff
@@ -84,13 +85,19 @@ func NewStatusSyncer(syncerLogger logr.Logger, syncTargetWorkspace logicalcluste
 			logger.V(2).Info("Set up downstream informer", "gvr", gvr.String())
 			return cache.ResourceEventHandlerFuncs{
 				AddFunc: func(obj interface{}) {
+					unstrob := obj.(*unstructured.Unstructured)
+					if hasDelayStatusSyncingAnnotation(unstrob) {
```

Comment:
This kind of looks like a band-aid - why isn't it?

Reply:
A band-aid?

Reply:
Why are we delaying the sync? Why is it for some period of time, as opposed to until some event occurs?